stable/opa/values.yaml

```diff
@@ -1,6 +1,10 @@
 # Default values for opa.
 # -----------------------
 #
+# Use "opa: false" to prevent opa from loading the below default bundles on launch.
+# When using this flag, a default configMap(opa-default-system-main) should be created as in the official documentation -
+# https://www.openpolicyagent.org/docs/v0.12.2/kubernetes-admission-control/#3-deploy-opa-on-top-of-kubernetes
+# opa: false
```

Comment:
Can you change the entire to comment to the following:

```
# The 'opa' key embeds an OPA configuration file. See https://www.openpolicyagent.org/docs/configuration.html for more details.
# Use 'opa: false' to disable the OPA configuration and rely on configmaps for policy loading. See https://www.openpolicyagent.org/docs/latest/kubernetes-admission-control/#3-deploy-opa-on-top-of-kubernetes and the `mgmt.configmapPolicies` section below for more details.
```

- need to provide more detail than just 'opa: false' because configmap policy loading is off-by-default.
- avoid linking to specific versions of the docs.

Reply:
Fair point.

Reply:
I've updated the PR.